### DISCUSSION TITLE
Connection to Jira Server

### DISCUSSION THREAD

**Discussion by LinaAMedina** [nbrown02/FlowViz-Jira#9] [Ideas]
Hi @nbrown02 , hope you are doing fine!!
I will love to implement the use of FlowViz through Jira in my Nestle teams and I am following the step by step you published but I am getting errors when trying to access the content of the powerbi
I am creating the API password and putting my email associated to Jira account but is not working :(

Can you please help me? 
(i tried to send a screenshot but I guess is not possible here)

**Comment by nbrown02** [comment]
Hey there,

Thanks for reaching out! If it helps, you can drop me an email at ***@***.*** and I can triage from there?

Thanks,

Nick
